charts/app-config/values-production.yaml

```diff
@@ -15,6 +16,64 @@ replicaCount: 3
 # Individual apps to be deployed by ArgoCD, and any values specific to those
 # apps.
 govukApplications:
+- name: asset-manager
+  chartPath: charts/asset-manager
+  helmValues:
+    workerEnabled: true
+    ingress:
+      enabled: true
+      annotations:
+        alb.ingress.kubernetes.io/load-balancer-name: assets-origin
+        alb.ingress.kubernetes.io/group.name: assets-origin
+        alb.ingress.kubernetes.io/group.order: '20'
+        alb.ingress.kubernetes.io/load-balancer-attributes: access_logs.s3.enabled=true,access_logs.s3.bucket=govuk-production-aws-logging,access_logs.s3.prefix=elb/assets-origin-eks
+        alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: >
+          [{"field":"host-header","hostHeaderConfig":{"values":["{{ .Values.assetsDomain }}"]}}]
+      hosts:
+      - name: assets-origin.eks.production.govuk.digital
+        path: /government/uploads/
+      - name: assets-origin.eks.production.govuk.digital
+        path: /media/
+    assetManagerNFS: &assets-nfs assets.blue.production.govuk-internal.digital
+    nginxConfigMap:
+      create: false
+      name: asset-manager-nginx-conf
+    extraEnv:
+      - name: ASSET_MANAGER_CLAMSCAN_PATH
+        value: /usr/bin/clamdscan
+      - name: GDS_SSO_OAUTH_ID
+        valueFrom:
+          secretKeyRef:
+            name: signon-app-asset-manager
+            key: oauth_id
+      - name: GDS_SSO_OAUTH_SECRET
+        valueFrom:
+          secretKeyRef:
+            name: signon-app-asset-manager
+            key: oauth_secret
+      - name: GOVUK_ASSET_ROOT  # testing in isolation
+        value: https://assets-origin.eks.production.govuk.digital
```

Comment:
I can't remember what this one does. Is it set right?

Reply:
I actually don't know what this does in the context of asset manager - it's used by plek, but asset manager doesn't generate links to static (I think ?🤷🏻‍♂️) 

https://github.com/alphagov/plek/blob/484196157989ba5cb385957538daf963f3bbf087/README.md?plain=1#L59

and it's not used within the Asset Manager codebase. So it can either get removed, or at least should be set to `assets.publishing.service.gov.uk`, instead of the internal domain.